daprdocs/content/en/reference/components-reference/supported-bindings/cloudflare-queues.md

```diff
@@ -0,0 +1,242 @@
+| `queueName` | Y | Output | Name of the existing Cloudflare Queue | `"mydaprqueue"`
+| `key` | Y | Output | Ed25519 private key, PEM-encoded | *See example above*
+| `cfAccountID` | Y/N | Output | Cloudflare account ID. Required to have Dapr manage the worker. | `"456789abcdef8b5588f3d134f74ac"def`
+| `cfAPIToken` | Y/N | Output | API token for Cloudflare. Required to have Dapr manage the Worker. | `"secret-key"`
+| `workerUrl` | Y/N | Output | URL of the Worker. Required if the Worker has been pre-provisioned outside of Dapr. | `"https://mydaprqueue.mydomain.workers.dev"`
+
+> When you configure Dapr to create your Worker for you, you may need to set a longer value for the `initTimeout` property of the component, to allow enough time for the Worker script to be deployed. For example: `initTimeout: "120s"`
+
+## Binding support
+
+This component supports **output binding** with the following operations:
+
+- `publish` (alias: `create`): Publish a message to the Queue.  
+  The data passed to the binding is used as-is for the body of the message published to the Queue.  
+  This operation does not accept any metadata property.
+
+## Create a Cloudflare Queue
+
+To use this component, you must have a Cloudflare Queue created in your Cloudflare account.
+
+You can create a new Queue in one of two ways:
+
+<!-- IGNORE_LINKS -->
+- Using the [Cloudflare dashboard](https://dash.cloudflare.com/)  
+- Using the [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/):
+  
+  ```sh
+  # Authenticate if needed with `npx wrangler login` first
+  npx wrangler queues create <NAME>
+  # For example: `npx wrangler queues create myqueue`
+  ```
+<!-- END_IGNORE -->
+
+## Configuring the Worker
+
+Because Cloudflare Queues can only be accessed by scripts running on Workers, Dapr needs to maintain a Worker to communicate with the Queue.
+
+Dapr can manage the Worker for you automatically, or you can pre-provision a Worker yourself. Pre-provisioning the Worker is the only supported option when running on [workerd](https://github.com/cloudflare/workerd).
+
+> **Important:** Use a separate Worker for each Dapr component. Do not use the same Worker script for different Cloudflare Queues bindings, and do not use the same Worker script for different Cloudflare components in Dapr (for example the Workers KV state store and the Queues binding).
+
+{{< tabs "Let Dapr manage the Worker" "Manually provision the Worker script" >}}
+
+{{% codetab %}}
+<!-- Let Dapr manage the Worker -->
+
+If you want to let Dapr manage the Worker for you, you will need to provide these 3 metadata options:
+
+<!-- IGNORE_LINKS -->
+- **`workerName`**: Name of the Worker script. This will be the first part of the URL of your Worker: for example, if the "workers.dev" domain configured for your Cloudflare account is `mydomain.workers.dev` and you set `workerName` to `mydaprqueue`, the Worker that Dapr deploys will be available at `https://mydaprqueue.mydomain.workers.dev`.
```

Comment:
```suggestion
- **`workerName`**: Name of the Worker script. This will be the first part of the URL of your Worker. For example, if the "workers.dev" domain configured for your Cloudflare account is `mydomain.workers.dev` and you set `workerName` to `mydaprqueue`, the Worker that Dapr deploys will be available at `https://mydaprqueue.mydomain.workers.dev`.
```